mathenger-api/src/main/java/com/example/mathengerapi/integrations/dspace/service/BotCommandsHandler.java

```diff
@@ -113,6 +114,18 @@ public void handleAllItemsOfCollection(Long chatId, UUID uuid) {
         botMessageService.send(message, chatId);
     }
 
+    public void handlePublication(Long chatId, UUID uuid) {
+        var informationAboutPublication = new StringBuilder("About this publication:\n\n");
+        Item work = dSpaceClient.getPublicationById(uuid);
+
+        String message = informationAboutPublication +
+                work.toString() +
+                "\n\nYou can view the publication " +
+                "by the link: http://localhost:4000/items/" +
```

Comment:
extract http://localhost:4000 to dSpaceUIBaseUrl field and inject it with @ Value annotation (https://www.baeldung.com/spring-value-annotation)

Reply:
The actual value of the variable should be in resources/application.properties file